MekHQ/src/mekhq/gui/adapter/ProcurementTableMouseAdapter.java

```diff
@@ -1,145 +1,131 @@
-            // **lets fill the pop up menu**//
-            // GM mode
-            menu = new JMenu("GM Mode");
+        // **lets fill the pop up menu**//
+        // GM mode
+        menu = new JMenu("GM Mode");
 
-            menuItem = new JMenuItem("Procure single item now");
-            menuItem.addActionListener(new AbstractAction() {
-                @Override
-                public void actionPerformed(ActionEvent e) {
-                    if (row < 0) {
-                        return;
-                    }
-                    if (oneSelected) {
-                        model.getAcquisition(row)
-                                .ifPresent(ProcurementTableMouseAdapter.this::tryProcureOneItem);
-                    } else {
-                        for (int curRow : rows) {
-                            if (curRow < 0) {
-                                continue;
-                            }
-                            model.getAcquisition(table.convertRowIndexToModel(curRow))
-                                    .ifPresent(ProcurementTableMouseAdapter.this::tryProcureOneItem);
-                        }
+        menuItem = new JMenuItem("Procure single item now");
+        menuItem.addActionListener(evt -> {
+            if (row < 0) {
+                return;
+            }
+            if (oneSelected) {
+                model.getAcquisition(row)
+                        .ifPresent(ProcurementTableMouseAdapter.this::tryProcureOneItem);
+            } else {
+                for (int curRow : rows) {
+                    if (curRow < 0) {
+                        continue;
                     }
+                    model.getAcquisition(table.convertRowIndexToModel(curRow))
+                            .ifPresent(ProcurementTableMouseAdapter.this::tryProcureOneItem);
                 }
-            });
-            menuItem.setEnabled(gui.getCampaign().isGM());
-            menu.add(menuItem);
-            menuItem = new JMenuItem("Procure all items now");
-            menuItem.addActionListener(new AbstractAction() {
-                @Override
-                public void actionPerformed(ActionEvent e) {
-                    if (row < 0) {
-                        return;
-                    }
-                    if (oneSelected) {
-                        model.getAcquisition(row)
-                                .ifPresent(ProcurementTableMouseAdapter.this::procureAllItems);
-                    } else {
-                        for (int curRow : rows) {
-                            if (curRow < 0) {
-                                continue;
-                            }
-                            model.getAcquisition(table.convertRowIndexToModel(curRow))
-                                    .ifPresent(ProcurementTableMouseAdapter.this::procureAllItems);
-                        }
+            }
+        });
+        menuItem.setEnabled(gui.getCampaign().isGM());
```

Comment:
Noticed when reading through, this can be removed as the whole thing only shows if they are a GM